src/sysinfo.cc

```diff
@@ -366,6 +366,30 @@ std::vector<CPUInfo::CacheInfo> GetCacheSizes() {
 #endif
 }
 
+std::string GetSystemName() {
+#if defined(BENCHMARK_OS_WINDOWS)
+  std::string str;
+  const unsigned COUNT =  32767;
```

Comment:
Again, why invent a magic number.
`MAX_COMPUTERNAME_LENGTH+1` ?
And that being said, isn't that a rather too big to have as an array on stack?
Maybe use resized `std::string`?